docs/source/api.rst

```diff
@@ -148,25 +149,33 @@ Generation
 **********
 
 LLMInterface
```

Comment:
I think the convention is the punctuation characters are as long as the text, shall we stick to this? https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Sure, I love conventions :-) Updated in https://github.com/neo4j/neo4j-genai-python/pull/102/commits/4b6590f87b8349daef58d6192a8967ef24dc0ff0